eng/jacoco-test-coverage/pom.xml

```diff
@@ -78,7 +78,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-identity</artifactId>
-      <version>1.5.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-identity;current} -->
+      <version>1.4.8</version> <!-- {x-version-update;com.azure:azure-communication-identity;current} -->
```

Comment:
Why a downgrade?

Reply:
This changes are done automatically as a part of patch release script and next version is picked from version_client file